product-list.css

```diff
@@ -0,0 +1,60 @@
+@import url(./variable.css);
+
+body {
+    background: var(--bg-color);
+}
+
+.filter_products {
+    flex-basis: 250px;
```

Comment:
any reason for using px here just want to know...

Reply:
> any reason for using px here just want to know...

No, I've changed it to rem. 🤔